static/css/main.css

```diff
@@ -9,7 +9,7 @@ body {
 }
 
 .navbar {
-  background: #5488c0;
+  background: red;
```

Comment:
keep consistent with previous code (hex color)

Reply:
Thanks for the tip! Picked a nice red hex for this! #f54242